src/components/Alert/Alert.VisualTests.stories.tsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import { Meta, Story } from '@storybook/react/types-6-0';
+import Alert, { AlertProps, AlertVariants } from './Alert';
+import Box from '../Box/Box';
+import Heading from '../Heading/Heading';
+
+export default {
+  title: 'Components/Alert/Visual Regression Tests',
+  component: Alert,
+} as Meta;
+
+const Template: Story<AlertProps> = args => (
+  <Box>
+    {AlertVariants.map(variant => (
+      <Alert
+        {...args}
+        className="m-bottom-md"
+        variant= {variant}
+      />
+    ))}
+    <Box className="m-top-xl m-bottom-md">
+      <Heading size="lg">
+       isCompact
+      </Heading>
+    </Box>
+    {AlertVariants.map(variant => (
+      <Alert
+        {...args}
+        className="m-bottom-md"
+        variant= {variant}
+        isCompact= {true}
+      />
+    ))}
+  </Box>
+);
+
+export const VariantsWithMessage = Template.bind({});
+VariantsWithMessage.args = {
+  message: 'Hello world!',
+}
+
+export const WithMessageWithIcon = Template.bind({});
```

Comment:
You can add a more human readable storyname like this for each component:

`WithMessageWithIcon.storyName = 'With Message, Icon";`

https://storybook.js.org/docs/react/api/csf#named-story-exports